docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,953 @@
+   You can change the interval if you like, just don't make it too long or you will have to wait a long time before you see your data!
+5. Let's add a tag to all your data so that you can query data from all your air quality sensors more easily in later steps.
+   In the **Tags** field, type `air-quality` and click **+ Tag: air-quality** when it appears to create a new tag.
+   This tag will now automatically be applied to all data collected by this data manager.
+6. Now the data management service is available to any components on your machine, and you can set up data capture on the sensor:
+7. On your **PM_sensor** card, click **Add method**.
+8. From the **Type** dropdown, select **Readings**.
+9. Set the **Frequency** to `0.1` readings per second.
+   This will capture air quality data once every ten seconds.
+   It is useful to capture data frequently for testing purposes, but you can always change this frequency later since you probably don't need to capture data this frequently all day forever.
+10. Save the config.
+
+### Create a fragment
+
+{{% alert title="Note" color="note" %}}
+If you are only using one air quality sensing machine for this tutorial, you do not need to create or use fragments, since fragments are useful only when configuring multiple machines.
+You can skip to [Test your sensors](#test-your-sensors).
+{{% /alert %}}
+
+While you configured your machine with the builder UI, the Viam app generated a JSON configuration file with all your parameters.
+This is the file that tells `viam-server` what resources are available to it and how everything is connected.
+Click **JSON** in the upper-left corner of the **CONFIGURE** tab to view the generated JSON file.
+You can manually edit this file instead of using the builder UI if you are familiar with JSON.
+
+In any case, now that the JSON is generated, you are ready to create a {{< glossary_tooltip term_id="fragment" text="fragment" >}}:
+
+1. Select and copy the entire contents of the JSON config.
+2. Navigate to the **FLEET** page and click [**Fragments**](https://app.viam.com/fragments) at the bottom of the left nav.
+3. Type in a name for your fragment, such as `air-sensing-machine` and click **Add fragment**.
+4. Replace the empty curly braces `{}` with the config you copied from your machine.
+5. Because the [Viam Agent](/fleet/provision/) config auto-populates into every machine's config, and configuring the agent using a fragment isn't supported, you do not need to include it in the fragment.
+
+   Delete the entire `agent_config` section including the comma just above it:
+
+   {{<imgproc src="/tutorials/air-quality-fleet/delete-agent-config.png" resize="x1100" declaredimensions=true alt="The section of the raw JSON that you should delete: the entire agent section." style="max-width:600px" >}}
+
+6. Click **Save fragment**.
+7. Now, you can actually delete the entire config from your machine!
```

Comment:
Just to be on the safe side
```suggestion
7. Now, you can actually delete the entire config from the machine you configured before!
```

Reply:
```suggestion
7. Now, you can actually delete the entire config from the machine you configured!
```
I think "before" is redundant with the past tense